src/content/docs/style-guide/writing-docs/article-templates/onramps-style-guide.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Onramps style guide
+contentType: page
+template: basicDoc
+topics:
+  - onramps
+---
+
+import apmApmSummaryOnrampsGif from 'images/apm_screenshot-full_apm-summary-onramps-gif.gif'
+
+import apmOnrampsIssuesExample from 'images/apm_screenshot-crop_onramps-issues-example.webp'
+
+import apmEmptyStateExample from 'images/apm_screenshot-crop_empty-state-example.webp'
+
+import apmActiveStateExample from 'images/apm_screenshot-crop_active-state-example.webp'
+
+import apmServiceLevelsExample from 'images/apm_screenshot-crop_service-levels-example.webp'
+
+
+Our onramps content is the help text that appears in the UI when you click the **?** icon. The purpose of in-product help is to help users understand the UI, interpret performance data, and New Relic solutions with their system problems. Onramps provide more detail than a [tooltip](https://one-core.newrelic.com/components/tooltip/). Ideally, they give users essential info so they can stay in the product and keep working, rather than switching over to read the docs. 
+
+<img
+    title=""
+    alt=""
+    src={apmApmSummaryOnrampsGif}
+/>
+
+## Content guidelines
```

Comment:
To me the entire doc reads like "Content guidelines." This section is about the heart of the content and what makes it really sing, whereas the rest of the doc is more like rules. I'd shoot for a heading that acknowledges that. What about something like, "What does a great on-ramp look like" or "Writing a great on-ramp" or similar.